### PR TITLE
Add type hint to _tableUnmodified method

### PR DESCRIPTION
**Pull request by bancer**

### CODE REVIEW

File: src/TestSuite/Fixture/ChecksumTestFixture.php [hunk 1]
```diff
@@ -80,10 +80,10 @@ public function drop(ConnectionInterface $db)
  * In all other cases where the initial and current hash differs, assume
  * the table has changed
  *
- * @param DboSource $db
+ * @param ConnectionInterface $db
```
Comment (ADmad) [inline src/TestSuite/Fixture/ChecksumTestFixture.php:83]: Should be FQCN

File: src/TestSuite/Fixture/ChecksumTestFixture.php [hunk 2]
```diff
@@ -80,10 +80,10 @@ public function drop(ConnectionInterface $db)
...
  * @return boolean
  */
-    protected function _tableUnmodified($db)
+    protected function _tableUnmodified(ConnectionInterface $db)
```
Comment (ADmad) [inline src/TestSuite/Fixture/ChecksumTestFixture.php:86]: Adding typehint strictly speaking would be non BC, so let's skip it. We can add it for new major Cake 4 compatible version of the plugin.

Comment (bancer) [reply src/TestSuite/Fixture/ChecksumTestFixture.php:86]: Well, all other methods in this class have type hints including the method that calls `_tableUnmodified`.

**Comment by ravage84** [comment]
Since the major version 2.0.0 which is compatible with CakePH p4.x is already released, changes to the API interfaces can't be  done, anymore.

@bancer feel free to create a new PR with anything left to be done.

**Pull request closed without merging**